plugins/org.eclipse.n4js/src/org/eclipse/n4js/postprocessing/ASTMetaInfoCache.java

```diff
@@ -109,9 +109,12 @@ public boolean hasBrokenAST() {
 			if (isCanceled()) {
 				return new Result<>(TypeRefsFactory.eINSTANCE.createUnknownTypeRef());
 			} else {
-				throw UtilN4.reportError(new IllegalStateException(
-						"cache miss: no actual type in cache for AST node: " + astNode
-								+ " in resource: " + resource.getURI()));
+				// TODO GH-73: Temporary hack. Reactivate the throw statement and solve cache miss problem, i.e. GH-95
+				Result<TypeRef> tr = new Result<>((TypeRef) astNode.eContainer());
+				return tr;
```

Comment:
Yes, https://github.com/eclipse/n4js/issues/95 blocks https://github.com/eclipse/n4js/issues/73 . We will definitely need to fix https://github.com/eclipse/n4js/issues/95 before being able to merge https://github.com/eclipse/n4js/issues/73